src/DetailView.php

```diff
@@ -281,8 +281,8 @@ public function render(): string
             strtr(
                 $this->template,
                 [
-                    '{attributes}' => Html::renderTagAttributes($this->attributes),
-                    '{containerAttributes}' => Html::renderTagAttributes($this->containerAttributes),
+                    '{attributes}' => Html::renderTagAttributes($this->properties),
+                    '{containerAttributes}' => Html::renderTagAttributes($this->containerProperties),
```

Comment:
This looks wrong. These are HTML tag attributes.

@vjik which properties/attributes did you mean?

Reply:
You're right. All HTML-related "attributes" must be keep.

Need to rename "attribute" that means "property of object": `DataField::$attribute` and related variables and methods.